core/src/main/java/com/github/jackchen/android/core/AndroidSample.kt

```diff
@@ -145,10 +144,17 @@ abstract class AndroidSample protected constructor() {
             for (i in 0 until sampleArray.length()) {
                 val sampleItem = SampleItem()
                 val sampleObject = sampleArray.getJSONObject(i)
-                sampleItem.className = sampleObject.optString("className")
-                sampleItem.title = sampleObject.getString("title")
-                sampleItem.desc = sampleObject.optString("desc")
-                sampleItem.path = sampleObject.optString("path")
+                sampleItem.className = sampleObject.getString("className")
+                val title = sampleObject.optString(SampleConstants.PARAMETER_TITLE)
+                if (title.isNotEmpty()) {
+                    sampleItem.title = title
+                } else {
+                    val simpleClassName = sampleItem.className.substring(
```

Comment:
className.substringAfterLast(".") equals to 

```
val simpleClassName = sampleItem.className.substring(className.lastIndexOf(".") + 1))
```

Reply:
Done